js/packages/cli/src/commands/mint.ts

```diff
@@ -399,7 +399,7 @@ export async function mintV2(
     (data.gatekeeper ? 33 : 0) +
     (data.gatekeeper?.expireOnUse ? 66 : 0);
 
-  log.info('Transaction size estimate: ', txnEstimate);
+  //log.info('Transaction size estimate: ', txnEstimate);
```

Comment:
Same here ! delete the code, but I think this log is quite important or debugging probably better put it as debug